torch/compiler/__init__.py

```diff
@@ -231,6 +284,7 @@ def is_compiling() -> bool:
     else:
         return _is_compiling_flag
 
+
```

Comment:
What's with the white space changes

Reply:
I removed `torch/compiler/__init__.py` from the `PYFMT` exclude pattern.